linux-gcc-aarch64.cmake

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2017, NeroBurner
+# All rights reserved.
+
+# install cross compiler on Ubuntu
+# - sudo apt install g++-arm-linux-gnueabihf
+# - with gfortran: sudo apt install gfortran-arm-linux-gnueabihf
+
+if(DEFINED POLLY_LINUX_GCC_AARCH64_)
+  return()
+else()
+  set(POLLY_LINUX_GCC_AARCH64_ 1)
+endif()
+
+
+# set system name, this sets the variable CMAKE_CROSSCOMPILING
+set(CMAKE_SYSTEM_NAME Linux)
+
+set(CROSS_COMPILE_TOOLCHAIN_PREFIX aarch64-linux-gnu)
+set(CROSS_COMPILE_TOOLCHAIN_SUFFIX -6)
+set(CROSS_COMPILE_TOOLCHAIN_PATH /usr/bin)
```

Comment:
Currently `CROSS_COMPILE_TOOLCHAIN_PATH` is used in `gcc-musl`/`arm-openwrt-linux-muslgnueabi` and set basing on environment variable that is tied to the toolchain name:
* https://github.com/ruslo/polly/blob/3b5ab745342adbff60ff6f9308bf55aabd19585b/gcc-musl.cmake#L19
* https://github.com/ruslo/polly/blob/3b5ab745342adbff60ff6f9308bf55aabd19585b/arm-openwrt-linux-muslgnueabi.cmake#L37

I think we should follow same approach here:
* Read environment variable `LINUX_GCC_AARCH64_ROOT`
* If not set use default path `/usr/bin` (do we need this?)
* Check path is correct, e.g. `$ENV{GCC_AARCH64_ROOT}/aarch64-linux-gnu-gcc-6` file exist
* If file not found emit error message with clear description of what is expected